Link GCk6 pages to Running Distributed guide (#2235)
grafana/k6-docs: docs/sources/k6/next/testing-guides/running-distributed-tests.md

For scenarios such as these, we've created the [k6-operator](https://github.com/grafana/k6-operator).

{{< admonition type="note" >}}

> See [Use the k6 Operator with Grafana Cloud k6](https://grafana.com/docs/k6/latest/set-up/set-up-distributed-k6/usage/k6-operator-to-gck6/) on how to integrate k6-operator with Grafana Cloud, including creation of a [Private Load Zone](https://grafana.com/docs/grafana-cloud/testing/k6/author-run/private-load-zone-v2/).

Reviewer: ```suggestion
Refer to [Use the k6 Operator with Grafana Cloud k6](https://grafana.com/docs/k6/latest/set-up/set-up-distributed-k6/usage/k6-operator-to-gck6/) on how to integrate the k6 Operator with Grafana Cloud, including creating [Private Load Zones](https://grafana.com/docs/grafana-cloud/testing/k6/author-run/private-load-zone-v2/).
```
